src/parsers/cdm_parser.jl

```diff
@@ -15,14 +15,16 @@ Returns:
     as values
 
 """
-function read_csv_data(file_path::String)
+function read_csv_data(file_path::String, baseMVA = 100.0)
```

Comment:
We shouldn't have a default value here. It might be convenient for most cases but the user should be aware of what the base is